pom.xml

```diff
@@ -79,9 +78,8 @@
         </dependency>
 
         <dependency>
-            <groupId>javax.validation</groupId>
-            <artifactId>validation-api</artifactId>
-            <version>${validation-version}</version>
+            <groupId>org.springframework.boot</groupId>
```

Comment:
why this changes ?

Reply:
`spring-boot-starter-validation` **automatically** checks the input when an API is called.